src/librustc_interface/interface.rs

```diff
@@ -61,6 +61,19 @@ impl Compiler {
     pub fn output_file(&self) -> &Option<PathBuf> {
         &self.output_file
     }
+    pub fn build_output_filenames(
```

Comment:
Please also add a newlinw between two functions here.

Reply:
The existing functions in `Compiler` are **not** separated by newlines, so I wonder if we should keep the style consistence? Or maybe all these functions should be separated by newlines?

Reply:
I didn't know they weren't separated. I think.it is better to keep it this way amd wait for a rustfmt pass of the file then.